tests/platform_tests/api/test_watchdog.py

```diff
@@ -122,9 +142,10 @@ def test_arm_different_timeout_greater(self, duthost, platform_api_conn, conf):
         actual_timeout_second = watchdog.arm(platform_api_conn, watchdog_timeout)
         remaining_time = watchdog.get_remaining_time(platform_api_conn)
         actual_timeout_second_second = watchdog.arm(platform_api_conn, watchdog_timeout_greater)
-
-        assert actual_timeout_second < actual_timeout_second_second
-        assert watchdog.get_remaining_time(platform_api_conn) > remaining_time
+        self.expect(actual_timeout_second < actual_timeout_second_second, "{}: 1st timeout {} should be smaller than 2nd timeout {}".format(test_arm_different_timeout_greater.__name__, actual_timeout_second, actual_timeout_second_second))
```

Comment:
These variable names are confusing (`actual_timeout_second`, `actual_timeout_second_second`) -- what does "second_second" mean? Can you create names that are more clear?

Also, please add units `seconds` after the second and third `{}`.